src/components/TransactionsTable/style.scss

```diff
@@ -2,63 +2,56 @@
 
 .transactions {
   .filters-row {
-    background: $gray-5;
-
     @include rem(margin-top, 61px);
   }
 
   .transaction-filters {
-    @include rem(padding, 0 76px 0 100px);
+    border-bottom: 1px solid $border-alt;
+    padding-bottom: 20px;
+
+    @include rem(margin, 0 100px);
 
     &__title {
       color: $heading;
       display: inline-block;
+      line-height: 1.35;
       font-weight: $fw-semi;
-      padding: 29px 0;
-
-      @include rem(font-size, 22px);
+      @include rem(font-size, 26px);
 
       @media all and (max-width: $screen-mobile-old) {
         display: block;
       }
     }
 
     &__item {
-      color: #636f7e;
+      color: #8c94a0;
```

Comment:
Why not to make this a constant variable in `LykkeExchangeWeb/src/styles/common/_variables.scss`?